src/myquadcopter_env.py

```diff
@@ -16,12 +16,13 @@
 from gazebo_connection import GazeboConnection
 from arm import armtakeoff
 
-#register the training environment in the gym as an available one
+# register the training environment in the gym as an available one
 reg = register(
     id='QuadcopterLiveShow-v0',
     entry_point='myquadcopter_env:QuadCopterEnv',
-    timestep_limit=100,
+    max_episode_steps=100,
```

Comment:
timestep_limit deprecated in favour of max_episode_steps in gym.

We should also change _step() to step()